static/app/utils/profiling/renderers/gridRenderer.tsx

```diff
@@ -4,16 +4,16 @@ import {FlamegraphTheme} from '../flamegraph/FlamegraphTheme';
 import {getContext, measureText, Rect} from '../gl/utils';
 
 export function getIntervalTimeAtX(configToPhysicalSpace: mat3, x: number): number {
-  const logicalToPhysical = mat3.fromScaling(
+  const logicalToPhysicalSpace = mat3.fromScaling(
     mat3.create(),
     vec2.fromValues(window.devicePixelRatio ?? 1, window.devicePixelRatio ?? 1)
   );
   const physicalToConfigSpace = mat3.invert(mat3.create(), configToPhysicalSpace);
 
   const logicalToConfigSpace = mat3.multiply(
     mat3.create(),
-    logicalToPhysical,
-    physicalToConfigSpace
+    physicalToConfigSpace,
+    logicalToPhysicalSpace
```

Comment:
I think we previously discussed this in https://github.com/getsentry/sentry/pull/31210#discussion_r789127642. Why did this flip again?

Reply:
It was actually broken when you pan around the chart and it worked in reverse. I think we need to investigate exactly why, maybe we are missing something here